src/lib/utils.js

```diff
@@ -90,3 +116,30 @@ export function sortBy (propertyName, desc = false) {
 export function asyncMap (array, asyncCallback) {
   return Promise.all(array.map(asyncCallback));
 }
+
+/**
+ * Create an array of numbers, starting from `start` and ending at `end`.
+ *
+ * @param {number} start - The number to start with.
+ * @param {number} end - The number to end with.
+ * @return Array<number>
+ */
+export function range (start, end) {
```

Comment:
**🎯 suggestion:** Maybe point out when we expect or output integers in place of numbers.